pkg/solana/logpoller/utils/anchor.go

```diff
@@ -0,0 +1,41 @@
+package utils
```

Comment:
Let's avoid a `utils` package. Maybe call it `discriminator.go` and leave it in the `logpoller` package for now?

Reply:
Yeah, originally the plan was to import this file from the private repo once it's made public, so I copied it over for now. But then @dhaidashenko pointed out this is the only method actually used, so I got rid of the others. I'll just move it into the main package.